lib/dx/loaders/dataloader.ex

```diff
@@ -1,6 +1,10 @@
 defmodule Dx.Loaders.Dataloader do
   @moduledoc """
   Uses `Dataloader` to load missing data incrementally.
+
+  ## Supported `loader_options` (passed to `dataloader`)
```

Comment:
Can you think of a better / nicer way to document this?
Typespec?

Reply:
In terms of documenting, this should be fine, I think. It would be good, however, to also mention `loader_options` in the `Dx` moduledoc as an option, linking here as the default implementation (or duplicating the nested options there).

Just an idea for formatting:

```suggestion
  ## Supported options
  
  These options are passed to `Dataloader.Ecto.new/2`:

```

I hope that hexdocs will link it accordingly.